docs/documentation/models/geometry/wing_tank_model.rst

```diff
@@ -0,0 +1,66 @@
+######################
+Wing tank volume model
+######################
+
+There are several models implemented for the computation of the volume of the tanks inside the wings of the aircraft.
+
+.. contents::
+
+.. _simple-model:
+
+*****************
+Simple tank model
+*****************
+
+The simple tank volume model is based on the assumption that the tank is a parallelepiped.
+
+The base of the tank is computed by making the assumptions that it stretches from the fuselage centerline to the wing span and that it is located between the 1st and 3rd stringer (respectively located at 30% and 60% of the chord).
+
+The height of the tank is computed using an average wing thickness obtained based on the thickness at the tip and root as well as a penalty factor which results in the following formula :
+
+.. math::
+    t_{avg} = 0.7 * \frac{c_{tip} * \left( t/c \right)_{tip} + c_{root} * \left( t/c \right)_{root}}{2}
+
+The volume is then computed with:
+
+.. math::
+    \text{MFV} = 0.3 * S_{w} * t_{avg}
+
+Finally the Maximum Fuel Weight (MFW) is computed based on the density of the fuel (which can be either Jet A1 or AvGas).
+
+This model is the one activated by default.
+
+.. _advanced_model
+
+*******************
+Advanced tank model
+*******************
+
+This model is based on the model presented in :cite:`jenkinson:2003` on page 65.
+
+The wing tank is discretized into sections. The start and end of the tank are defined by a ratio of the wing half-span left as an user input. The areas of the tank cross-sections are then computed at a given number of stations in the spanwise direction and finally integrated to give the tank volume. A top view of the tank is shown in Figure `1`_ .
+
+.. image:: ../../../img/wing_tank_model_graph.svg
+  :width: 600
+  :name: 1
+
+The width of each section is based on the wing chord minus some space at the leading edge and the trailing edge, highlighted in red in Figures `1`_ and `2`_. The distance between the wing leading edge and the leading edge of the tank is defined by a ratio of the chord (assumed constant at all stations) which is left as a user input. The distance between the trailing edge of the wing and that of the tank is computed accounting for the flaps and the aileron as well as an added margin. The lengths of the flaps and ailerons are computed based on a ratio of the chord also assumed constant regardless of the span. This is illustrated in Figures `1`_ and `2`_.
```

Comment:
```suggestion
The width of each section is based on the wing chord minus some space at the leading edge and the trailing edge, highlighted in red in Figures `1`_ and `2`_. The distance between the wing leading edge and the leading edge of the tank is defined by a ratio of the chord (assumed constant at all stations) which is left as a user input. The distance between the trailing edge of the wing and that of the tank is computed accounting for the flaps and the aileron (whichever has the largest chord ratio) as well as an added margin. The width of the flaps and ailerons are computed based on a ratio of the chord also assumed constant regardless of the span. This is illustrated in Figures `1`_ and `2`_.
```

Reply:
I suggest "width of the flap" instead of "length", to remain consistent with the mentioned "width of each section", where width refers to the chordwise direction. "Length" would rather refer to the spanwise direction.

Reply:
I agree, rephrased in e124776. Although I wonder if we should be even more specific and say _the chordwise direction_ or _the spanwise direction_. It might make it too heavy though